operations/patch_module.go

```diff
@@ -48,8 +48,14 @@ func PatchSetModule() cli.Command {
 				return errors.Wrap(err, "can't test for uncommitted changes")
 			}
 
-			if (!uncommittedOk && !conf.UncommittedChanges) && uncommittedChanges {
-				grip.Infof("Uncommitted changes are omitted from patches by default.\nUse the '--%s, -u' flag or set 'patch_uncommitted_changes: true' in your ~/.evergreen.yml file to include uncommitted changes.", uncommittedChangesFlag)
+			useMbox := true
+			if uncommittedChanges {
+				if uncommittedOk || conf.UncommittedChanges {
+					grip.Info("Patches with uncommitted changes cannot be enqueued on a commit queue")
```

Comment:
Nit: should end sentence with punctuation since this is logged to the user

Reply:
done